utils/utils_test.go

```diff
@@ -53,3 +53,41 @@ func TestGetGithubBaseURL(t *testing.T) {
 		})
 	}
 }
+
+func TestIsGoFile(t *testing.T) {
+	tests := []struct {
+		got  string
+		want bool
+	}{
+		{"foo.go", true},
+		{"bar.gol", false},
+		{"foobar", false},
+		{"fubar.g", false},
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("given the filename %s", tt.got), func(t *testing.T) {
+			got := IsGoFile(tt.got)
+			assert.Equal(t, got, tt.want)
```

Comment:
The assertion is inverted
```suggestion
			assert.Equal(t, tt.want, got)
```